index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <div id="main">
+      <h1 id="title">Nelson Rolihlahla Mandela</h1>
+      <div id="img-div">
+        <img id="image" src="Nelson-Mandela-South-African.jpg" alt="Nelson Rolihlahla Mandela">
+        <h4 id="img-caption">Our hero, Nelson Mandela</h4>
+      </div>
```

Comment:
Please put the divs in section tags as required, to indicate the different sections in the html file.

Reply:
- [ ] Kindly make sure every time you explain why the changes are suggested. And why not giving some `hint` about how to fix the issue :smile: